codegenerator/cli/npm/envio/package.json

```diff
@@ -19,6 +19,9 @@
     "url": "https://github.com/enviodev/hyperindex/issues"
   },
   "homepage": "https://envio.dev",
+  "engines": {
+    "node": ">=22.0.0 <=22.10.0"
+  },
```

Comment:
I'm worried about this, why do we need <=22.10 again? I use v24 on my machine for eg